config/taxonomy/codegen.spec.yaml

```diff
@@ -34,3 +34,31 @@ paths:
                 $ref: "../../charts/fybrik/files/taxonomy/policy_manager_response.json"
         '400':
           description: Invalid status value
+
+  /getDatasetInfo:
+      post:
+        summary: getDatasetInfo.
+        parameters:
+          - in: header
+            name: X-Request-DataCatalog-Cred
+            schema:
+              type: string
+            required: true
+        requestBody:
+          description: Data Catalog Request Object.
+          required: true
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: "../../charts/fybrik/files/taxonomy/data_catalog_request.json"
+        responses:
+          '200':
+            description: successful operation
+            content:
+              application/json:
+                schema:
+                  type: object
+                  $ref: "../../charts/fybrik/files/taxonomy/data_catalog_response.json"
+          '400':
+            description: Invalid status value
```

Comment:
I believe the description should indicate that an error happened, 400 is a valid status value.

Reply:
fixed. Now we have polcymanager.spec.yaml and catalogconnector.spec.yaml to separate out the APIs. I have fixed the description in the  catalogconnector.spec.yaml.